src-gui/src/renderer/rpc.ts

```
@@ -417,29 +434,163 @@ export async function getMoneroAddresses(): Promise<GetMoneroAddressesResponse>
+  accept: T,
 ): Promise<void> {
   try {
     await invoke<ResolveApprovalArgs, ResolveApprovalResponse>(
       "resolve_approval_request",
-      { request_id: requestId, accept },
+      { request_id: requestId, accept: accept as object },
     );
   } catch (error) {
-    // Refresh approval list when resolve fails to keep UI in sync
-    await refreshApprovals();
     throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)